Code/Legacy/CryCommon/IMovieSystem.h

```diff
@@ -826,7 +826,15 @@ struct IAnimSequence
         eSeqFlags_DisplayAsFramesOrSeconds = BIT(18), //!< Display Start/End time as frames or seconds
     };
 
-    virtual ~IAnimSequence() {};
+    IAnimSequence()
+    {
+        AZ_Info("IAnimSequence", "IAnimSequence");
```

Comment:
add anim sequence name in the log (not sure why we want to track that, but would be more useful with more info)

Reply:
Also, this kind of message shouldn't be an info log, it's more of a trace log (`AZ_Trace`).